File: src/test/java/org/tindalos/principle/domain/checker/TestFixture.java
package org.tindalos.principle.domain.checker;

import org.tindalos.principle.domain.core.logging.ScalaLogger;
import org.tindalos.principle.domain.core.logging.TheLogger;

public class TestFixture {
    
    public static void setLogger() {
        TheLogger.setLogger(new ScalaLogger() {
            public void info(String msg) {
                System.out.println(msg);
            }

            public void error(String msg) {
                System.err.println(msg);
            }
        });
    }

}

File: src/test/java/org/tindalos/principle/domain/checker/TestFixture.scala
package org.tindalos.principle.domain.checker

import org.tindalos.principle.domain.core.logging.TheLogger
import org.tindalos.principle.domain.core.logging.ScalaLogger

object TestFixture {

  def setLogger() = {
    TheLogger.setLogger(new ScalaLogger() {
      override def info(msg: String) = {
        System.out.println(msg)
      }

      override def error(msg: String) = {
        System.err.println(msg)
      }
    })
  }

}